src/ray/gcs/gcs_server/gcs_actor_manager.h

```diff
@@ -267,16 +294,28 @@ class GcsActorManager : public rpc::ActorInfoHandler {
   /// again.
   void ReconstructActor(const ActorID &actor_id, bool need_reschedule = true);
 
-  /// Callbacks of actor registration requests that are not yet flushed.
-  /// This map is used to filter duplicated messages from a Driver/Worker caused by some
-  /// network problems.
+  /// Callbacks of actor registration requests.
```

Comment:
```suggestion
  /// Callbacks of pending `RegisterActor` requests.
```
Explicitly put `RegisterActor` here can help readers understand it.